src/Parallel-Solvers/Parallel-Explicit/Example_Yaml_Scripts/example_sgh_opt_region.yaml

```diff
@@ -106,6 +106,13 @@ optimization_options:
     optimization_process: topology_optimization
     optimization_objective: minimize_kinetic_energy
     use_solve_checkpoints: true
+    rol_params:
```

Comment:
This looks great. Thanks for doing this yaml parsing for this.  Will make users happy.